internal/impl/gcp/cache_cloud_storage.go

```diff
@@ -15,7 +15,9 @@ func gcpCloudStorageCacheConfig() *service.ConfigSpec {
 		Summary(`Use a Google Cloud Storage bucket as a cache.`).
 		Description(`It is not possible to atomically upload cloud storage objects exclusively when the target does not already exist, therefore this cache is not suitable for deduplication.`).
 		Field(service.NewStringField("bucket").
-			Description("The Google Cloud Storage bucket to store items in."))
+			Description("The Google Cloud Storage bucket to store items in.")).
+		Field(service.NewStringField("content_type").
+			Description("Optional field to explicitly set the Content-Type.").Default("").Optional())
```

Comment:
I guess I can get rid of `.Default("")` when also using `.Optional()` but I wanted to make sure. What do you say?

Reply:
Hey @kabukky, yeah that's correct, all you need is optional.

Reply:
Thanks! I've changed it now.